Apps/Database/src/Models/DelegateInvitation.cs

```diff
@@ -51,11 +51,31 @@ public class DelegateInvitation : AuditableEntity
         public DelegateInvitationStatus Status { get; set; }
 
         /// <summary>
-        /// Gets or sets the sharing code.
+        /// Gets or sets the sharing code hash function.
         /// </summary>
         [Required]
-        [MaxLength(6)]
-        public string SharingCode { get; set; } = null!;
+        [MaxLength(50)]
+        public string SharingCodeHashFunction { get; set; } = null!;
```

Comment:
It might be preferable to leave this as an enum type and specify how to convert it in GatewayDbContext.

Reply:
Changed to enum and added code table.